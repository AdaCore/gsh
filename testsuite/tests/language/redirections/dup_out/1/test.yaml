title: "Test of >& redirection operator"
