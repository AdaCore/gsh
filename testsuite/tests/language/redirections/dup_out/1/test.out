== content of error.out ==
error
==
== content of file5.out ==
file5
==
